snuba/migrations/snuba_migrations/metrics/0021_polymorphic_bucket_materialized_views.py

```diff
@@ -36,6 +36,7 @@ def forwards_local(self) -> Sequence[operations.SqlOperation]:
                     "countState(values_rows) as count"
                 ),
                 metric_type="distribution",
+                materialization_version=3,
```

Comment:
@onewland I think this also goes against the rule then? But splitting up that function seems quite extreme.

Reply:
I am happy to do whatever y'all think should be done. I personally think shared modules like `templates.py` don't make a ton of sense to have in the first place if we want to avoid accidentally breaking old migrations at cost of DRY.